examples/windows/windows-perfmon-proc-owner.ps1

```diff
@@ -0,0 +1,44 @@
+$computername = "DESKTOP-26GUTUU"
+$colProcs = Get-wmiobject win32_process -computername $computername  -Credential $credential | select *, @{Name = "Owner"; Expression = { ($_.GetOwner()).User } }
+$colPerfs = Get-wmiobject win32_perfformatteddata_perfproc_process -computername $computername  -Credential $credential
```

Comment:
please update these to `Get-CimInstance` to align with future direction of MSFT and reduce load of query against system

Reply:
as for win32_perfformatteddata_perfproc_process I can replace this with Get-CimInstance, the first one does not provide the owner when changing from Get-WmiObject to Get-CimInstance